backend/local_app/kiva/statistics.py

```diff
@@ -1,4 +1,25 @@
-from django.db import connection
+from django.db import connection, models
 
 from .models import Loan, Lender
 
+class LoanStats(object):
+    '''Performs the following raw SQL queries on the Loans model:
+    1. get_all_loans selects all rows and columns from the Loans table 
+    '''
+    
+    def get_all_loans(self):
```

Comment:
I'm not sure there is any need for this to be a function, it could just be a variable that holds the SQL statement. Let me know what you all think, maybe there is a reason to use a function.